bin/prestart.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+# Refresh all config files in order
+#
+# This script is typically called once at the container start, but
+# it can be called manually if WP config details must be changed
+
+# The database and memcached config files are separate to avoid collisions
+# if their backends' onChange handlers are triggered simultaneously
+echo "******running onstart script*********"
+
+until [[ `curl -s ${CONSUL}:8500/v1/health/state/passing | grep mysql-primary`  ]]
+do
+  echo "mysql-primary not healthly...."
+  sleep 5
+done
+
+until [[ `curl -s ${CONSUL}:8500/v1/health/state/passing | grep nfs`  ]]
```

Comment:
I tried to use the more specific api call that you dug up for these "waiting for services" functions, but I found this syntax to be much simpler. Using /v1/health/service/$SERVICE_NAME?passing call I would have to speficially check for the "no cluster leader" condition when consul isn't fully started and also check to see if the service was currently passing.

Using this method I just grab the list of all passing services and grep for the service name I am looking for. grep returns true if the service name is found and false if it is not. Thishandles consul not being fully started and the service itself being down in a single condition in the if statement.

I'll take a second look at the more specific api call, I believe I was in the middle of debugging something else when I discovered that this wasn't working as expected and decided to revert back to this api call.